riscv/encoding.h

```diff
@@ -1033,6 +1033,10 @@
 #define MASK_FCVT_H_L  0xfff0007f
 #define MATCH_FCVT_H_LU 0xd4300053
 #define MASK_FCVT_H_LU  0xfff0007f
+#define MATCH_POLLENTROPY 0xf1500073
+#define MASK_POLLENTROPY  0xfffff07f
+#define MATCH_GETNOISE 0x7a900073
+#define MASK_GETNOISE  0xfffff07f
```

Comment:
This commit adds these defines, then the following commit removes them. If they shouldn't be there then we should fix that here in the first commit, instead of adding a second commit to correct this one.